docs/source/benchmarking/industry/doap.rst

```diff
@@ -1,5 +1,5 @@
 The Description of a Project (DOAP) vocabulary
```

Comment:
convert `The Description of a Project (DOAP) vocabulary` to `The Description of a Project vocabulary (DOAP) `